### PR TITLE
fix credits for Stockfish

### PR DESCRIPTION
**Pull request by jalpp**
add credits for Lichess/stockfish.js/Stockfish and freechess

### CODE REVIEW

File: frontend/next.config.mjs
Comment (jackstenglein) [inline frontend/next.config.mjs:1]: Can you revert the change to this file please?

**Comment by jalpp** [comment]
Will do
